### DISCUSSION TITLE
The create performance of Hadoop nnbench is low

### DISCUSSION THREAD

**Discussion by jyialive** [juicedata/juicefs#3653] [General]
使用hadoop nnbench测试时create性能，juicefs的性能比较差，meta用的tikv，但是其他的也差不多。

**Comment by tangyoupeng** [comment]
Can you use juicefs bench command test first

**Comment by jyialive** [comment]
> Can you use juicefs bench command test first

Is there any difference between hadoop nnbench and juicefs bench? mulit thread？ I think the problem is the concurrent ‘jfs_create’ performance，Conflicting resources inside

**Comment by tangyoupeng** [comment]
Could you provide the command?

**Comment by jyialive** [comment]
> Could you provide the command?

./hadoop jar ../share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient-3.3.4-tests.jar nnbench -operation create_write -maps 100 -numberOfFiles 10000 -blockSize 1048576 -bytesToWrite 0 -baseDir jfs://myjfs/xxx

**Comment by tangyoupeng** [comment]
性能比较差是和什么做对比的

**Comment by jyialive** [comment]
跟hdfs namenode比

**Comment by tangyoupeng** [comment]
有可能，使用tikv create的时候会存在多个网络来回
